CONTRIBUTING.md

```diff
@@ -397,8 +397,11 @@ If running in interactive mode (without `-y|--yes`), the script will output a
 summary of pending changes and ask for confirmation before executing the
 actions.
 
-Once this is done, a package will be automatically created and published from CI
-to PyPI.
+Once this is done, a tag will be created on the repository. 
+You should then create a Github
```

Comment:
What about [creating the release](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) as a draft, so that we need only edit it a bit? It's a nice touch with one curl invocation :)